docs-site/content/27.0/api/search.md

```diff
@@ -223,7 +223,7 @@ When a `string[]` field is queried, the `highlights` structure will include the
Shoe` and not documents with a `category` field set as `Shoe Rack`. <br><br>Using the `:` (non-exact) operator will do a word-level partial match on the field, without taking token position into account (so is usually faster). Eg: `category:Shoe` will match records with `category` of `Shoe` or `Shoe Rack` or `Outdoor Shoe`. <br><br>Tip: If you have a field that doesn't have any spaces in the values across any documents and want to filter on it, you want to use the `:` operator to improve performance, since it will avoid doing token position checks. <br><br>**Escaping Special Characters:**<br>You can also filter using multiple values and use the backtick character to denote a string literal: <code>category:= [\`Running Shoes, Men\`, \`Sneaker (Men)\`, Boots]</code>.<br><br>**Negation:**<br>Not equals / negation is supported via the `:!=` operator, e.g. `author:!=JK Rowling` or `id:!=[id1, id2]`. You can also negate multiple values: `author:!=[JK Rowling, Gilbert Patten]`<br><br> To exclude results that _contains_ a specific string during filtering you can do `artist:! Jackson` will exclude all documents whose `artist` field value contains the word `jackson`.  <br><br>**Numeric Filtering:**<br>Filter documents with numeric values between a min and max value, using the range operator `[min..max]` or using simple comparison operators `>`, `>=` `<`, `<=`, `=`. <br><br>You can enable `"range_index": true` on the numerical field schema for fast range queries (will incur additional memory usage for the index though). <br><br>Examples:<br>-`num_employees:<40`<br>-`num_employees:[10..100]`<br>-`num_employees:[<10, >100]`<br>-`num_employees:[10..100, 140]` (Filter docs where value is between 10 to 100 or exactly 140).<br>-`num_employees:!= [10, 100, 140]` (Filter docs where value is **NOT** 10, 100 or 140).<br><br>**Multiple Conditions:**<br>You can separate multiple conditions with the `&&` operator.<br><br>Examples:<br>- `num_employees:>100 && country: [USA, UK]`<br>- `categories:=Shoes && categories:=Outdoor`<br><br>To do ORs across _different_ fields (eg: color is blue OR category is Shoe), you can use the ` ||` operator. <br><br>Examples:<br>- `color: blue || category: shoe`<br>- `(color: blue || category: shoe) && in_stock: true` <br><br>**Filtering Arrays:**<br>filter_by can be used with array fields as well. <br><br>For eg: If `genres` is a `string[]` field: <br><br>- `genres:=[Rock, Pop]` will return documents where the `genres` array field contains `Rock OR Pop`. <br>- `genres:=Rock && genres:=Acoustic` will return documents where the `genres` array field contains both `Rock AND Acoustic`. <br><br>**Prefix filtering:**<br> You can filter on records that begin with a given prefix string like this: <br><br> `company_name: Acm*` <br><br> This will will return documents with names that begin with `acm`, for e.g. a name like `Acme Corp.`. <br><br>**Geo Filtering:**<br> Read more about [GeoSearch and filtering](geosearch.md) in this dedicated section. |
```

Comment:
This diff seems to have removed the sections about `Embedding Filters in API Keys` and also some additional details about prefix filtering. 

Could you add those back?

Reply:
Resolved in e3ae45614176fb603c9c323d2cabe0634c8e6bb1